ChangeLog

```diff
@@ -208,6 +564,99 @@
+  - Made a bunch of functions const
+  - Got rid of an iterator loop
+  - Remove unimplemented function `void print_land_stats();`
+- Test website binaries on Travis.
+- Improve autoformat PR detection and diff info (#3701)
+  - Check whether the branch has an open PR *after* running fix_formatting. If the user opens the PR within three minutes after pushing, this allows the action to push the changes straight to the new PR, so no "dummy" commit is needed in this case to get the action running.
+  - Fix `git diff` showing empty in non-PR branches
+- Fix some compiler warnings (#3694)
+- Catch exception when bzr returns non-zero exit status (#3714): When the bzr command fails to detect a branch the version string correctly falls back to REVDETECT-BROKEN-PLEASE-REPORT-THIS instead of crashing the script
+- Fix translations formatting (#3722)
+  - Add command line switches to `fix_formatting.py` for formatting selectively
+  - Add formatting directives to translation update script for generated Lua files
+- Removing the gce environment setting so build should on hyper-v (with more ram?)
+- Let fix_formatting.py format the codecheck directory (#3723)
+  - Add codecheck directory to pyformat runs
+  - Give codecheck rules file ending so that fix_formatting will pick them up.
+  - Format codecheck rules.
+- Fix sigsev for XDG_DATA_DIRS (#3700)
+- Launchpad mirror (#3766)
+  - Mirrors all changes at widelands:master to lp:widelands
+  - Link entries in xdg to debian to fix daily PPA
+  - Merge latest Debian rules from https://sources.debian.org/src/widelands/1:20-2/debian/rules/ to fix daily PPA
+  - Fix appdata/desktop filenames
+  - Fix Debian build errors about missing files
+  - Only list missing and delete symlinks for debian packaging.
+  - Fix CRLF/LF handling in LP commit messages
+  - Copy rather than move icons and appdata for debian packaging
+- Prettier stdout files when regression_test is run with python 3 (#3800): Print regression test suite output as raw byte strings
+- Fix stalling test suite scenarios (#3784)
+  - `stable_save` now uses last save time and average frame rate for waiting
+  - pass desired speed explicitly to `stable_save`
+- Update CMakeLists for Ubuntu Focal
+- Replace dependency python with python3
+- Fix packaging for Focal and #3783
+- Fix travis MacOS builds (#3848): On Travis MacOS, gettext needs to be added to the PATH and some CMake CXX and linker flags have to be set because brew force-linking is no longer allowed for whatever reason
+- Fix travis failure on gcc-9.1_debug (#3861)
+- Improve SDL2 detection on modern systems (#3872)
+- Add Git hash to AppVeyor and bzr version numbers (#3894)
+- updated innosetup file to reflect new Icu filenames (#3903)
+- gcc-10 compatibility (#3908)
+- Replaced all uses of boost::bind with lambda functions and added a codecheck rule to deprecate boost::bind() and boost::ref()
+- Added missing DLLs to appveyor builds (#3917)
+- Appveyor cleanup (#3919)
```

Comment:
I would remove the above 2 entries as they are not related top widelands but to MSYS2 changes. If you want to have them you could write various fixes to the appveyor environment